addons/sale_project/models/project_project.py

```diff
@@ -141,14 +141,20 @@ def action_view_sols(self):
 
     def action_view_sos(self):
         self.ensure_one()
-        all_sale_orders = self._fetch_sale_order_items({'project.task': [('is_closed', '=', False)]}).order_id
+        all_sale_orders = self._fetch_sale_order_items({'project.task': [('is_closed', '=', False)]})._filter_access_rules_python('read').order_id
```

Comment:
Hum, good to know, I guess we probably have something to do inside `_fetch_sale_order_items` 🧐

Reply:
Should I change something then ?